zerver/tests/test_queue_worker.py

```diff
@@ -107,7 +105,8 @@ def test_slow_queries_worker(self) -> None:
         self.assertEqual(args[2], "stream")
         self.assertEqual(args[3], "errors")
         self.assertEqual(args[4], "testserver: slow queries")
-        self.assertEqual(args[5], "    test query (data)\n    second test query (data)\n")
+        self.assertEqual(args[5], '    127.0.0.1       GET     200 -1000ms (db: 1ms/13q)'
```

Comment:
We should avoid testing for specific timings—this [failed CI on master](https://circleci.com/gh/zulip/zulip/30546) when the query took `4ms` instead of `1ms`.